### DISCUSSION TITLE
Free screenshot Function

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#12686] [Q&A]
I want to implement the function of freely selecting screenshots in Avalonia. Has anyone done similar work?Hope for help, thank you.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/3a3473f2-3973-47be-b159-ae265a73c4f0)

**Comment by timunie** [comment]
Check out AdornerLayer and Adorner in general.

**Reply by pyking-wang** [reply]
Is there an example of AdornerLayer in Avalonia, which seems to be different from the use of Adorner in WPF

**Reply by timunie** [reply]
iirc there are some examples in Avalonia.Behaviors samples
